core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/numeric/NumberFunctionExpression.kt

```diff
@@ -0,0 +1,21 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.numeric
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.operator.FunctionOperator
+import ch.ergon.dope.validtype.NumberType
+
+sealed class NumberFunctionExpression(
+    private val symbol: String,
+    private val value: TypeExpression<NumberType>,
+    private val extraValue: TypeExpression<NumberType>? = null,
+) : TypeExpression<NumberType>, FunctionOperator {
+    override fun toDopeQuery(): DopeQuery {
+        val valueDopeQuery = value.toDopeQuery()
+        val extraValueDopeQuery = extraValue?.toDopeQuery()
+        return DopeQuery(
+            queryString = toFunctionQueryString(symbol = symbol, valueDopeQuery, extra = extraValueDopeQuery),
```

Comment:
can you remove completely this _toFunctionQueryString_-method (with the extra-param) and extend the other _toFunctionQueryString_ with a _filterNotNull_?:
```
    fun toFunctionQueryString(symbol: String, vararg arguments: DopeQuery?) = arguments.filterNotNull().joinToString(
        ", ",
        prefix = "$symbol(",
        postfix = ")",
    ) {
        it.queryString
    }
```

Reply:
done